.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "twig-language-2.indentStyle": "space",
+    "twig-language.indentStyle": "space"
+}
```

Comment:
your .vscode file should not be versionned

Reply:
ok deleted